packages/osh/osh.pacscript

```diff
@@ -0,0 +1,37 @@
+name="osh"
+pkgdir="${STOWDIR}/${name}"
+version="0.9.5"
+description="Oil Shell - A bash-compatible UNIX shell written in Python"
+url="https://www.oilshell.org/download/oil-${version}.tar.xz"
+gives="oil"
+breaks="${name}-bin ${name}-git ${name}-deb ${name}-app"
+hash="d89d8c081c8bc536dfc7b437b965a265c07e24cf5142ff26b21b4cc4a3139e32"
+
+# Edit MAKEFLAGS here
+MAKEFLAGS=
+
+prepare() {
+  mkdir -p tmp
+
+  # HACK: Change `TMP` directory to `./tmp/` for building
+  # We do not want this to expand
+  # shellcheck disable=SC2016
+  sed 's/TMP=${TMPDIR:-\/tmp}/TMP=.\/tmp/' -i "configure"
```

Comment:
TMPDIR=./tmp/ is a lot more reliable than this

Reply:
Well, that works but when testing I found out that the `Makefile` makes a directory called `_tmp`, where apparently no file is stored throughout the whole build. So, is it okay to point `TMP` to that directory?

Reply:
I'm just saying you shouldn't use sed to change the value of the variable.  You should just use `TMPDIR=whatever ./configure`.

This kind of sed usage is bad because it can easily break later, and is unnecessary